ui_advanced_config.py

```diff
@@ -545,7 +549,10 @@ def init_entry_widgets(self):
         # OBJECT
```

Comment:
there is a problem updating the id of an object. the object associated with the old id is still in the json and a duplicated with the new id is added in addition to the old object. the combobox does not include this duplicate.

Reply:
will do tomorrow

Reply:
Fixed I think

Reply:
Old id seems to now be replaced with new ID, looks good